msteams-platform/get-started/build-basic-tab-app.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Build your basic tab app
+description: Build your basic tab app
+ms.localizationpriority: high
+ms.topic: reference
+---
+# Build your basic tab app
+
+Start Microsoft Teams app development with your first Teams tab app.
+
+:::image type="content" border="false" source="../assets/images/get-started/get-started-tab.png" alt-text="Screenshot shows you the conceptual tab app in Teams client.":::
+
+In this tutorial, get familiarized with a basic Teams tab app in one of the following ways:
+
+* **Teams Toolkit codespaces**: The codespace instance allows you to experience a Teams app almost instantaneously. It opens Visual Studio Code, where the Teams Toolkit extension, the app source code, and all dependencies are pre-packaged for you.
+* **Step-by-step guide**: Allows you to create a Teams app from scratch with all development environment setup.
+
+# [Teams Toolkit codespaces](#tab/teamstoolkitcodespaces)
+
+Before you create your codespace, ensure that you have:
+
+* A GitHub account to create your codespace instance.
+* A [Microsoft 365 account](https://developer.microsoft.com/microsoft-365/dev-program) with sideloading permission.
+* A [Microsoft 365 tenant](../concepts/build-and-test/prepare-your-o365-tenant.md)
+
+> [!TIP]
+> [GitHub codespaces](https://github.com/features/codespaces) offers the free plan up to a fixed amount of usage every month. If you need to free up the space, go to [github.com/codespaces](https://github.com/codespaces) and delete the codespaces that you no longer need.
+
+To create a Teams tab app with codespaces, follow the steps:
+
+1. Select the following button to open codespaces.
+
+    <a href="https://github.com/codespaces/new?hide_repo_select=true&ref=v3&repo=348288141&machine=standardLinux32gb&location=WestUs2&devcontainer_path=.devcontainer%2Fhello-world-tab-codespaces%2Fdevcontainer.json&resume=1" target="_blank"><img src="https://github.com/codespaces/badge.svg" alt="Open hello-world tab in GitHub Codespaces"></a>
+
+   You may be asked to sign in to GitHub account if you haven't.
+
+1. Select **Create codespaces**.
+
+   :::image type="content" source="../assets/images/get-started/codespace.png" alt-text="Screenshot shows you the GitHub page to create a codespace for tab."lightbox="../assets/images/get-started/codespace.png":::
```

Comment:
The image for codespace creation is not correct:
With **resume=1** setting in the URL, the UI should looks like:

![image](https://github.com/MicrosoftDocs/msteams-docs/assets/10163840/5cb792c5-c41f-4c5b-b75d-ff18e641de9c)

User can just click `Create new Codespace` with default options, or click the `Change options` to customize the creation.

Reply:
updated the screenshot as suggested.